optimum/intel/openvino/modeling.py

```diff
@@ -451,6 +443,11 @@ def _from_transformers(
         else:
             ov_config = OVConfig(dtype="fp32")
 
+        if is_sentence_transformers_available():
+            library_name = "sentence_transformers"
```

Comment:
I do not think that it is good idea to switch model export library by default if lib available. Model can be not supported by sentence transformers. It is better to have this logic in specific class and keep export part in base model without changes.